offer-manager-domain/src/main/groovy/life/qbic/business/offers/content/CreateOfferContent.groovy

```diff
@@ -138,7 +138,7 @@ class CreateOfferContent implements CreateOfferContentInput, FetchOfferOutput{
             return 0
         } else {
             return offerItems.sum {
-                it.quantity * it.unitPrice * overheadRatio
+                (it.quantity * it.unitPrice -it.quantityDiscount) * overheadRatio
```

Comment:
same here, we should use the already calculated item net

Reply:
We need to calculate them for the specific groups. We only have them for all items.

Reply:
ah you mean the totalCosts?

Reply:
yes

Reply:
this property exists for every item